src/c_api.h

```diff
@@ -75,9 +75,9 @@ static void **PyGEOS_API;
  * PyCapsule_Import will set an exception on error.
  */
 static int
-import_pygeos_c_api(void)
+import_shapely_c_api(void)
 {
-    PyGEOS_API = (void **)PyCapsule_Import("pygeos.lib._C_API", 0);
+    PyGEOS_API = (void **)PyCapsule_Import("shapely.lib._C_API", 0);
```

Comment:
This was the crucial change to have our cython code actually pick up shapely instead of pygeos if it happens to be co-installed